src/config/serverConfigDefaults.ts

```diff
@@ -42,6 +42,7 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     isoformOverrideSource: 'uniprot',
     show_hotspot: true,
     show_oncokb: true,
+    show_oncokb_cbiocosmic: true,
```

Comment:
Update property names after update of backend branch.

Reply:
fixed